mirdata/annotations.py

```diff
@@ -79,6 +79,26 @@ def __repr__(self):
         return repr_str
 
 
+class MultiAnnotator(Annotation):
+    """Multiple annotator class.
+    This class should be used for datasets with multiple annotators (e.g. multiple annotators per track).
+
+    Attributes:
```

Comment:
as I'm thinkign about this more, I think the annotation type should be passed as an argument to init, and used in the validation. In the future I think it will save us a lot of code.